### PR TITLE
OSDOCS-9505 updated create cluster command

### PR DESCRIPTION
**Pull request by xJustin**
Updated the `rosa create cluster` command to add the operator roles

Version(s):

4.13+
Issue:

https://issues.redhat.com/browse/OSDOCS-9505

Link to docs preview:

https://71023--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly#rosa-hcp-sts-creating-a-cluster-cli_rosa-hcp-sts-creating-a-cluster-quickly

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by dfitzmau** [comment]
Hi @xJustin . Please tick the QE box as Ying Zhang apprioved the PR.

File: modules/rosa-hcp-sts-creating-a-cluster-cli.adoc [hunk 1]
```diff
@@ -52,11 +52,11 @@ $ rosa create cluster --private --cluster-name=<cluster_name> \
     --sts --mode=auto --hosted-cp --subnet-ids=<private-subnet-id>
 ----
 
-** If you used variables like `OIDC_ID` and `SUBNET_IDS`, you can use those references when creating your cluster. For example, run the following command:
+** If you used the `OIDC_ID`, `SUBNET_IDS`, and `OPERATOR_ROLES_PREFIX` variables, you can use those references when creating your cluster. For example, run the following command:
```
Comment (dfitzmau) [inline modules/rosa-hcp-sts-creating-a-cluster-cli.adoc:55]:
```suggestion
** If you used the `OIDC_ID`, `SUBNET_IDS`, and `OPERATOR_ROLES_PREFIX` variables to x, you can continue to use these variables when creating your cluster. For example, run the following command:
```

I added `x` in teh update as I cannot determine where these variables might have been previously used.

**Review by dfitzmau — COMMENTED** [review]
Hi @xJustin .

Nice work· I added some comments inline.

/remove-label peer-review-in-progress

/label peer-review-done

**Comment by xJustin** [comment]
Thanks @dfitzmau !

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

**Pull request merged**